### DISCUSSION TITLE
Multiscale Backbone

### DISCUSSION THREAD

**Discussion by marceliten** [facebookresearch/detectron2#3066] [Q&A]
Is it correct that from the backbone CNN not just from the last layer proposals are generated, but from several positions from the backbone proposals are generated to access different scales?

**Comment by GitHub-Crow** [comment]
the first **from the last layer** is normal backbone like  `Resnet50, 101`, the second **from several positions** is `FPN`, more detail can found in this [paper](https://arxiv.org/abs/1612.03144)
